grafana/dashboards/default_dashboard.json

```
@@ -563,7 +488,7 @@
         "y": 9
       },
       "hiddenSeries": false,
-      "id": 8,
+      "id": 11,§
```

```suggestion
      "id": 11,
```
That's probably the `Â` char that was causing some errors

---

yes, thats it. Thanks!